docs/customization_guide/inference_protocols.md

```diff
@@ -114,6 +114,54 @@ These options can be used to configure the KeepAlive settings:
 
 For client-side documentation, see [Client-Side GRPC KeepAlive](https://github.com/triton-inference-server/client/blob/main/README.md#grpc-keepalive).
 
+#### Limit Endpoint Access (BETA)
+
+In some use cases, Triton users may want to restrict the access of the protocols on a given endpoint.
+For example, there can be need for two separate protocol groups that one exposes standard inference
+protocols for user access, while the other one exposes other extension protocols for administration
+usage and should not be accessible by non-admin user.
+
+The following option can be specified to declare an restricted protocol group:
+
+```
+--grpc-restricted-protocol=<protocol_1>,<protocol_2>,...:<restricted-header>=<restricted-key>
```

Comment:
would the HTTP API be affected as well?

assuming it would not be. How would you expect the HTTP API to be affected?

trying to wrap my head around `--grpc-restricted-protocol=<protocol_1>,<protocol_2>`. 

could this be `--grpc-restricted-protocol=grpc,http`? If yes, then why the prefix `--grpc-` and not use `--restricted-protocol` as the option name?